### DISCUSSION TITLE
Access scopes updation

### DISCUSSION THREAD

**Discussion by TNagendra** [kinngh/shopify-node-express-mongodb-app#133] [Q&A] [answered]
How to update access scopes? I want to add write_orders access scope. Thanks in advance.

**Comment by kinngh** [comment, marked as answer]
Add it to your `.env`. 

- If you're on the latest version, after updating your `env` run `npm run update:config` and restart your server.
- If you're not on the latest version, update your `env` and restart your server.

After doing either, you need to go through the auth process again, which is as easy as going into admin and clicking on your app again.
